github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -658,6 +658,47 @@ presubmits:
           resources:
             requests:
               memory: "4Gi"
+
+  - name: pull-kubevirt-gosec
+    cluster: ibm-prow-jobs
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
```

Comment:
can you add the `"rehearsal.allowed": "true"` annotation? we then will be able to rehearse this job to see how it's working.

Reply:
done